tests/foreman/ui/test_dashboard.py

```diff
@@ -631,12 +632,22 @@ def test_positive_content_host_subscription_status(self):
                 set_context(session, org=org.name)
                 self.assertTrue(self.dashboard.validate_chss_navigation(
                     'Invalid', u'subscription_status = invalid'))
-                self.assertIsNotNone(self.dashboard.wait_until_element(
-                    common_locators['kt_search_no_results']))
-                self.assertTrue(self.dashboard.validate_chss_navigation(
-                    'Partial', u'subscription_status = partial'))
-                self.assertIsNotNone(self.dashboard.wait_until_element(
-                    common_locators['kt_search_no_results']))
+                if not bz_bug_is_open(1482150):
+                    self.assertIsNotNone(self.dashboard.wait_until_element(
+                        common_locators['kt_search_no_results']))
+                    self.assertTrue(self.dashboard.validate_chss_navigation(
+                        'Partial', u'subscription_status = partial'))
+                    self.assertIsNotNone(self.dashboard.wait_until_element(
+                        common_locators['kt_search_no_results']))
+                else:
+                    hosts = self.dashboard.find_elements(
```

Comment:
In normal situation when there is a content hots self.dashboard.find_elements(self.contenthost._search_locator() % "") will return all content hosts web elements?